### PR TITLE
Added support for network encryption key (GTK) set to border router

### PR DESCRIPTION
**Pull request by mikaleppanen**
Setting enables to set GTK0 on border router to fixed value for testing purposes.

### CODE REVIEW

**Comment by mikaleppanen** [comment]
@mikter please review

File: configs/Wisun_Stm_s2lp_RF_lab.json [hunk 1]
```diff
@@ -44,6 +44,7 @@
         "uc-fixed-channel": "0xffff",
         "bc-fixed-channel": "0xffff",
         "network-name": "\"ARM-WS-LAB-NWK\"",
+        "GTK0": null,
```
Comment (mikter) [inline configs/Wisun_Stm_s2lp_RF_lab.json]: Do we need these defaults be present here?

Comment (mikaleppanen) [reply configs/Wisun_Stm_s2lp_RF_lab.json]: Removed nulls from .json files, added GTK0 setting to readme.

**Comment by mikaleppanen** [comment]
@mikter updated, please review again.

**Comment by mikaleppanen** [comment]
Updated readme. @mikter please review the changes.

**Pull request merged**